src/inspector/node_string.cc

```diff
@@ -126,20 +80,18 @@ String fromUTF16(const uint16_t* data, size_t length) {
   return String(buffer.out(), utf8_length);
 }
 
-const uint8_t* CharactersUTF8(const std::string_view s) {
+const uint8_t* StringUtil::CharactersUTF8(const std::string_view s) {
   return reinterpret_cast<const uint8_t*>(s.data());
 }
 
-size_t CharacterCount(const std::string_view s) {
+size_t StringUtil::CharacterCount(const std::string_view s) {
   // The utf32_length_from_utf8 function calls count_utf8.
   // The count_utf8 function counts the number of code points
   // (characters) in the string, assuming that the string is valid Unicode.
   // TODO(@anonrig): Test to make sure CharacterCount returns correctly.
```

Comment:
Does this todo hold, or can we remove it?

Reply:
I can remove it in a follow up PR. I'd like to limit the scope of this big PR.

Reply:
Sounds good to me.